datadog_checks_base/datadog_checks/base/checks/libs/prometheus.py

```diff
@@ -0,0 +1,85 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from prometheus_client.metrics_core import Metric
+from prometheus_client.parser import _parse_sample, _replace_help_escaping
+
+
+def text_fd_to_metric_families(fd):
```

Comment:
Could be nice to have unit tests specific to this function, that will help understand what is expected.

Reply:
It's relatively well covered. I won't do that here at least.